src/app/dynamic-forms/cinchy-dynamic-forms.component.ts

```diff
@@ -9,36 +9,41 @@ import {
-import {MatDialog} from '@angular/material';
-import {CinchyService, QueryType} from '@cinchy-co/angular-sdk';
-import {Form, IForm} from './models/cinchy-form.model';
-import {DropdownDataset} from './service/cinchy-dropdown-dataset/cinchy-dropdown-dataset';
-import {FormSection} from './models/cinchy-form-sections.model';
-import {CinchyColumn, ICinchyColumn} from './models/cinchy-column.model';
-import {IQuery} from './models/cinchy-query.model';
-import {FormField} from './models/cinchy-form-field.model';
-import {ResponseType} from './enums/response-type.enum';
-import {EventCallback, IEventCallback} from './models/cinchy-event-callback.model';
-import {ToastrService} from 'ngx-toastr';
-import {NgxSpinnerService} from 'ngx-spinner';
-import {AppStateService} from "../services/app-state.service";
-import {takeUntil} from "rxjs/operators";
-import {Subject} from "rxjs";
-import {CinchyQueryService} from "../services/cinchy-query.service";
-import {MessageDialogComponent} from "./message-dialog/message-dialog.component";
-import {PrintService} from "./service/print/print.service";
+import { isNullOrUndefined } from 'util';
+import { ChildFormDirective } from './fields/cinchy-child-form.directive';
+import { MatDialog } from '@angular/material';
+import { CinchyService, QueryType } from '@cinchy-co/angular-sdk';
+import { Form, IForm } from './models/cinchy-form.model';
+import { DropdownDataset } from './service/cinchy-dropdown-dataset/cinchy-dropdown-dataset';
+import { FormSection } from './models/cinchy-form-sections.model';
+import { CinchyColumn, ICinchyColumn } from './models/cinchy-column.model';
+import { IQuery } from './models/cinchy-query.model';
+import { FormField } from './models/cinchy-form-field.model';
+import { ResponseType } from './enums/response-type.enum';
+import { EventCallback, IEventCallback } from './models/cinchy-event-callback.model';
+import { ToastrService } from 'ngx-toastr';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { AppStateService } from '../services/app-state.service';
+import { takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+import { CinchyQueryService } from '../services/cinchy-query.service';
+import { MessageDialogComponent } from './message-dialog/message-dialog.component';
+import { PrintService } from './service/print/print.service';
 
 @Component({
-  selector: 'cinchy-dynamic-forms',
-  templateUrl: './cinchy-dynamic-forms.component.html',
-  styleUrls: ['./style/style.scss'],
+  selector     : 'cinchy-dynamic-forms',
+  templateUrl  : './cinchy-dynamic-forms.component.html',
+  styleUrls    : ['./style/style.scss'],
   encapsulation: ViewEncapsulation.None
 })
 export class CinchyDynamicFormsComponent implements OnInit, OnChanges, OnDestroy {
   @Input() RowId: number | string;
   @Input() FormId: number | string;
+  nonExpandedSection: boolean;
+  closedSectionData: boolean = false;
+  openSectionData: boolean = false;
+  saveBtnInd: boolean = false;
```

Comment:
can we rename this variable to something that makes more sense? "enableSaveBtn" or something

Reply:
Resolved